packages/module/src/providers/blockfrost.provider.ts

```
@@ -114,76 +88,50 @@ export class BlockfrostProvider implements IFetcher, ISubmitter {
+
+    return {
+      coinsPerUTxOSize: bfResponse.coins_per_utxo_word || '',
+      collateralPercent: bfResponse.collateral_percent || 0,
+      decentralisation: bfResponse.decentralisation_param,
+      epoch: bfResponse.epoch,
+      keyDeposit: bfResponse.key_deposit,
+      maxBlockExMem: bfResponse.max_block_ex_mem || '',
+      maxBlockExSteps: bfResponse.max_block_ex_steps || '',
+      maxBlockHeaderSize: bfResponse.max_block_header_size,
+      maxBlockSize: bfResponse.max_block_size,
+      maxCollateralInputs: bfResponse.max_collateral_inputs || 0,
+      maxTxExMem: bfResponse.max_tx_ex_mem || '',
+      maxTxExSteps: bfResponse.max_tx_ex_steps || '',
+      maxTxSize: bfResponse.max_tx_size,
+      maxValSize: bfResponse.max_val_size || '',
+      minFeeA: bfResponse.min_fee_a,
+      minFeeB: bfResponse.min_fee_b,
+      minPoolCost: bfResponse.min_pool_cost,
+      poolDeposit: bfResponse.pool_deposit,
+      priceMem: bfResponse.price_mem || 0,
+      priceStep: bfResponse.price_step || 0,
+    };
   }
 
   async submitTx(tx: string): Promise<string> {
-    try {
-      const headers = { 'Content-Type': 'application/cbor' };
-      const { data, status } = await this._axiosInstance.post(
-        'tx/submit', toBytes(tx), { headers },
-      );
-
-      if (status === 200)
-        return data;
-
-      throw parseHttpError(data);
-    } catch (error) {
-      throw parseHttpError(error);
-    }
+    const txBytes = toBytes(tx);
+    const bfResponse = await this.blockfrostAPI.txSubmit(txBytes);
+
+    return bfResponse;
   }
 
   private async fetchPlutusScriptCBOR(scriptHash: string): Promise<string> {
-    const { data, status } = await this._axiosInstance.get(
-      `scripts/${scriptHash}/cbor`,
-    );
-
-    if (status === 200)
-      return data.cbor;
+    const bfResponse = await this.blockfrostAPI.scriptsCbor(scriptHash);
 
-    throw parseHttpError(data);
+    return bfResponse.cbor || '';
```

silent fail if there is no `cbor` field (or is with falsy value)